tests/queries/0_stateless/03210_variant_with_aggregate_function_type.sql

```diff
@@ -8,7 +8,7 @@ CREATE TABLE source
 
 ) ENGINE = MergeTree ORDER BY ();
 
-INSERT INTO source SELECT ['fail', 'success'][number % 2] as Name, number AS Value FROM numbers(1000);
+INSERT INTO source SELECT ['fail', 'success'][((number + 1) % 2) + 1] as Name, number AS Value FROM numbers(1000);
```

Comment:
Just small fix - arrow index starts from 1, not from 0, so value 'success' did not used before.